storage/replica_raftstorage.go

```diff
@@ -112,16 +114,32 @@ func (r *Replica) Entries(lo, hi, maxBytes uint64) ([]raftpb.Entry, error) {
 	// If neither the number of entries nor the size limitations had an
 	// effect, we weren't able to supply everything the client wanted.
 	if len(ents) != int(hi-lo) && (maxBytes == 0 || size < maxBytes) {
```

Comment:
I don't quite understand maxBytes; so if a call comes in from `lo` to `hi`, where `lo` (and some number of entries after it) has already been compacted, but there are more than maxBytes worth of entries remaining in between `lo` and `hi`, it's still a valid call?